src/docs/product/performance/trace-view.mdx

```diff
@@ -36,11 +36,13 @@ The transaction view provides a split view similar to the [span view](/product/p
 
 ### Orphan Traces and Broken Subtraces
 In a fully instrumented system, a span in one service will connect to a transaction in a subsequent service. For a variety of reasons, a transaction in a service may not arrive in Sentry. When Sentry encounters these types of transactions within a trace, we link the transactions with a dashed line since they can no longer be directly connected to the root.
+In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. In this case, the error will not appear on the trace view and the quick trace won't appear on the Issue Details page.
```

Comment:
Do we want a shoutout to the "Open in Discover" button here?

Reply:
Hmm, I'm not sure I understand why?

Reply:
Because errors that are part of a missing transaction will not appear in trace view but can be found in discover.